quiz_app_questions.json

```
@@ -35,9 +35,9 @@
 		"answer": "Motherboard"
 	},
 	{
-		"question": "Which of these is not a peripheral, in computer terms?",
-		"options": [ "Keyboard", "Mouse", "Motherboard", "Monitor" ],
-		"answer": "Motherboard"
+		"question": "The first alpha version of Flutter was released in ?",
```

Would you please remove the extra space?

```suggestion
		"question": "The first alpha version of Flutter was released in?",
```

---

Have removed the space. Sorry for the mistake.